src/entry_point.py

```diff
@@ -136,9 +144,66 @@ def __call__(self, parser, namespace, values, option_string=None):
         "in the environment specified by --prefix",
     )
 
+    subcommands = p.add_subparsers(dest="command")
+    uninstall_subcommand = subcommands.add_parser(
+        "uninstall",
+        description="Uninstalls a conda directory and all environments inside the directory.",
+    )
+    uninstall_subcommand.add_argument(
+        "--prefix",
+        action="store",
+        required=True,
+        help="Path to the conda directory to uninstall.",
+    )
+    uninstall_subcommand.add_argument(
+        "--remove-caches",
+        action="store_true",
+        required=False,
+        help=(
+            "Removes the notices cache and runs conda --clean --all to clean package caches"
+            " outside the installation directory."
+            " This is especially useful when pkgs_dirs is set in a .condarc file."
+            " Not recommended with multiple conda installations when softlinks are enabled."
+        ),
+    )
+    uninstall_subcommand.add_argument(
+        "--remove-config-files",
+        choices=["user", "system", "all"],
+        default=None,
+        required=False,
+        help=(
+            "Removes all .condarc files."
+            " `user` removes the files inside the current user's"
+            " home directory and `system` removes all files outside of that directory."
+            " Not recommended when multiple conda installations are on the system"
+            " or when running on an environments directory."
+        ),
+    )
+    uninstall_subcommand.add_argument(
+        "--remove-user-data",
+        action="store_true",
+        required=False,
+        help=(
+            "Removes the ~/.conda directory."
+            " Not recommended when multiple conda installations are on the system"
+            " or when running on an environments directory."
+        ),
+    )
+
     args, args_unknown = p.parse_known_args()
 
-    args.prefix = os.path.abspath(args.prefix)
+    if args.command != "uninstall":
+        group_args = getattr(g, "_group_actions")
+        if all(getattr(args, arg.dest) is None for arg in group_args):
```

Comment:
```suggestion
        if all(getattr(args, arg.dest, None) is None for arg in group_args):
```

default param missing?